src/index.ts

```diff
@@ -43,7 +43,7 @@ const visitor = (preNode: hast.Element): void => {
 const selector = (node: hast.Element): boolean => node.tagName === 'pre'
 
 
-interface Options {
+export interface Options {
```

Comment:
It is better to rename Options to RehypePrismOptions, which can avoid naming conflicts as much as possible, if we export the interface.

Reply:
Makes sense! Corrected.